package.json

```diff
@@ -503,6 +513,18 @@
           "group": "navigation",
           "when": "debugType == amalgamator"
         }
+      ],
+      "editor/lineNumber/context": [
+        {
+          "command": "cdt.debug.enableHWBreakpoint",
+          "group": "navigation",
+          "when": "(!enableHardware && !disableHardware) || enableHardware"
```

Comment:
Having `enableHardware` twice in this expression is unexpected.

Reply:
The solution has been changed to use `Toggle Breakpoint`